packages/micro/src/lib/handler.ts

```diff
@@ -0,0 +1,30 @@
+// Utilities
+import { logError } from './error';
+
+export const handle = async (file: string) => {
+  let mod: unknown;
+
+  try {
+    mod = file.endsWith('.mjs') ? await import(file) : await require(file); // Await to support exporting Promises
```

Comment:
Heads up: I ended up reverting this from the other PR just to make sure we had time to test it out here before the major release.

Reply:
I've updated to match the latest main commit. I think it can be done in a different PR.